lib/components/app/app-menu.tsx

```diff
@@ -180,11 +180,13 @@ class AppMenu extends Component<
     return (
       <>
         <div
+          aria-expanded={isPaneOpen}
           aria-label={
             isPaneOpen
               ? intl.formatMessage({ id: 'components.AppMenu.closeMenu' })
               : intl.formatMessage({ id: 'components.AppMenu.openMenu' })
           }
+          aria-roledescription="button"
```

Comment:
The best thing would be to convert the `<div>` into a `<button>` and remove the extra roles.